ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-commands.sh

```diff
@@ -217,13 +217,10 @@ Resources:
         FromPort: 5678
         ToPort: 5678
         CidrIp: 0.0.0.0/0
-      - IpProtocol: tcp
-        FromPort: 6443
```

Comment:
I think this will break non-metal tests, as they dont port forward anything and the apiserver listens here with no configurable option.

Reply:
Ah, I didn't realize this was used everywhere, I'll fix that in the update where I remove the code to force using my dev branch in the microshift repo.

Reply:
fixed